conda_forge_tick/migrators/nvtools.py

```diff
@@ -0,0 +1,244 @@
+import copy
+import logging
+import os.path
+from typing import Any
+
+import conda_forge_tick.migrators
+from conda_forge_tick.contexts import ClonedFeedstockContext
+from conda_forge_tick.migrators_types import AttrsTypedDict, MigrationUidTypedDict
+from conda_forge_tick.utils import (
+    get_bot_run_url,
+    yaml_safe_dump,
+    yaml_safe_load,
+)
+
+
+def _file_contains(filename, string):
+    with open(filename) as f:
+        return string in f.read()
+
+
+def _insert_requirements_build(meta):
+    insert_before: bool = True
+    before: list[str] = []
+    after: list[str] = []
+    requirements_found = False
+    build_found = False
+    with open(meta) as f:
+        for line in f:
+            if build_found:
+                insert_before = False
+            elif line.startswith("requirements"):
```

Comment:
```suggestion
            if line.startswith("requirements"):
```

Reply:
This was intentional. Once you find the requirements.build section, you don't need to do any more searching for it because you already know where to split the text. Maybe I could refactor to not execute insert_before=False, but I'm not sure it's worth the effort?